src/VirtoCommerce.Platform.Web/web.config

```diff
@@ -16,9 +16,9 @@
       </environmentVariables>
     </aspNetCore>
     <security>
-      <requestFiltering>
+      <requestFiltering allowDoubleEscaping="true">
```

Comment:
But this will work only for "in-process" hosting mode, can you check how it will work for "outofprocess" mode?
And I think these changes can significantly affect all API and required full API regress tests. I think before applying this fix we need to get more know-how this option `allowDoubleEscaping`  works.